packages/js/manifests/wrap/src/formats/wrap.info/deserialize.ts

```diff
@@ -29,7 +29,7 @@ export function deserializeWrapManifest(
   }
 
   if (!options || !options.noValidate) {
```

Comment:
Should this be && instead of ||?

Reply:
Nope it should be or basically it is if (options is falsy or options.novalidate is falsy)